api/jobimp/v1/jobimp.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package api.jobimp.v1;
+
+option go_package = "api/jobimp/v1";
```

Comment:
nit: We use the full package [elsewhere](https://github.com/openmarketplaceengine/openmarketplaceengine/blob/83eee0882b6c4996af598ef97d1619a43957f215/omeapi/location/v1beta1/location.proto#L10)
```
option go_package = "github.com/openmarketplaceengine/openmarketplaceengine/internal/omeapi/location/v1beta1";
```